public/scripts.js

```diff
@@ -0,0 +1,81 @@
+const boxArray = [];
+let i = 1;
+
+const onChange = (e, i) => {
+  i = i - 1;
+
+  if (e.target.checked) {
+    const someStringToCheck = e.currentTarget.id;
+    const indexOfSpace = someStringToCheck.indexOf("-");
+    const index = parseInt(
+      someStringToCheck.substr(indexOfSpace + 1, someStringToCheck.length + 1)
+    );
+    document.getElementById(`box-${i}`);
+    let heightValue = document.getElementById(`height-${index}`).value + "px";
+    let WidthValue = document.getElementById(`width-${index}`).value + "px";
```

Comment:
staraj się zachować konwencje we własnym kodzie. Zmienna `heightValue` jest tego samego typu co ta `WidthValue`, a mają różną konwencje nazewniczą. Jedno jest od dużej litery a drugie od małej. Do zmiennych obecnie w JS najpopularniejszą konwencją jest camelCase gdzie zmienne zaczynamy od małej litery a każde kolejne "słowo" od dużej. Czyli `heightValue` jak najbardziej. Konwencje camelCase od dużej pierwszej litery używamy w przypadku klas lub interfejsów (w przypadku type scripta). np. `class HeightValue { }`